package.json

```diff
@@ -65,7 +65,7 @@
     "gmusic-mini-player.js": "^2.0.10",
     "gmusic-theme.js": "^2.1.15",
     "gmusic-ui.js": "^2.2.3",
-    "gmusic.js": "^6.1.1",
+    "gmusic.js": "^6.1.0",
```

Comment:
gmusic.js@6.1.0 uses slider to get current playback position. Next gmusic.js versions use audio element instead (for now) and updating slider has no sense then